src/sentry/migrations/0323_backfill_ownership_for_projects_with_codeowners.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.28 on 2022-10-04 23:32
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_project_ownership(apps, schema_editor):
+    ProjectOwnership = apps.get_model("sentry", "ProjectOwnership")
+    ProjectCodeOwners = apps.get_model("sentry", "ProjectCodeOwners")
+
+    for codeowner in RangeQuerySetWrapperWithProgressBar(ProjectCodeOwners.objects.all()):
+        if ProjectOwnership.objects.filter(project_id=codeowner.project_id).exists():
+            continue
+
+        ownership = ProjectOwnership(
+            project_id=codeowner.project_id,
+            auto_assignment=False,
+            suspect_committer_auto_assignment=False,
+        )
+        ownership.save()
```

Comment:
```
ProjectOwnership.objects.get_or_create(
    project_id=codeowner.project_id,
    defaults={"auto_assignment": False, "suspect_committer_auto_assignment": False},
)
```

Then you can also remove the `exists` check earlier since this will handle it safely.